Resources/Prototypes/Reagents/chemicals.yml

```diff
@@ -81,6 +81,66 @@
         - !type:ReagentThreshold
           min: 5
 
+- type: reagent
+  id: Benzene
+  name: reagent-name-benzene
+  desc: reagent-desc-benzene
+  physicalDesc: reagent-physical-desc-acidic
+  color: "#E7EA91"
+  boilingPoint: 353.2
+  meltingPoint: 278.7
+  metabolisms:
+    Poison:
+      effects:
+      - !type:HealthChange
+        damage:
+          types:
+            Cellular: 1 #it's a carcinogen, reckon this is fine
+
+- type: reagent
+  id: Hydroxide
+  name: reagent-name-hydroxide
+  desc: reagent-desc-hydroxide
+  physicalDesc: reagent-physical-desc-alkaline
+  flavor: alkaline
+  color: "white"
+  boilingPoint: 1661.0
+  meltingPoint: 596.0
+
+- type: reagent
+  id: SodiumHydroxide
+  name: reagent-name-sodium-hydroxide
+  desc: reagent-desc-sodium-hydroxide
+  physicalDesc: reagent-physical-desc-alkaline
+  flavor: alkaline
+  color: "white"
+  boilingPoint: 1661.0
+  meltingPoint: 596.0
+  metabolisms:
+    Poison:
+      effects:
+      - !type:HealthChange
+        damage:
+          types:
+            Heat: 1.5
+    Medicine:
+      effects:
+      - !type:ChemVomit
+        conditions:
+        - !type:ReagentThreshold
+          min: 5
+        probability: 0.1
+
+- type: reagent
+  id: Fersilicite
```

Comment:
Could you list a few future planned uses you have for fersilicite?

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/149967078/c4368bbb-b9e4-4706-8179-2801183598ed)

Reply:
I agree, but could you list like two or three just to make sure we're not adding a chem that will be unused?

Reply:
Leporazine is definitely one, probably space lube, and I'll definitely work on some more meds, buff chemicals were discussed, so maybe those too